### PR TITLE
release first web service demo

### PR DESCRIPTION
**Pull request by michael-msx**
1 Create ipcs_service module
2 update ipcs_parent pom.xml  for Spring version 4.1.6
3 build separate  ipcs-service.war 

demo URL path http://localhost:8080/ipcs-service/loginService?user=Person&pwd=11
result : true or false

### CODE REVIEW

File: .idea/libraries/Maven__xml_apis_xml_apis_1_0_b2.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<component name="libraryTable">
```
Comment (seeyou8286) [inline .idea/libraries/Maven__xml_apis_xml_apis_1_0_b2.xml]: Can move these xml to ipcs_service?

File: .idea/libraries/Maven__org_springframework_spring_webmvc_4_1_6_RELEASE.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<component name="libraryTable">
```
Comment (seeyou8286) [inline .idea/libraries/Maven__org_springframework_spring_webmvc_4_1_6_RELEASE.xml]: Can move these xml to ipcs_service?

File: .idea/libraries/Maven__antlr_antlr_2_7_7.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<component name="libraryTable">
```
Comment (seeyou8286) [inline .idea/libraries/Maven__antlr_antlr_2_7_7.xml]: Can move these xml to ipcs_service?

**Comment by michael-msx** [comment]
had delete .idea folder

**Comment by seeyou8286** [comment]
Looks ok

**Pull request merged**